visualization/dashboard/src/MLIDashboard/Controls/DatasetExplorerTab/DatasetExplorerTab.tsx

```diff
@@ -188,7 +199,7 @@ export class DatasetExplorerTab extends React.PureComponent<IDatasetExplorerTabP
                             </Callout>}
                             <AccessibleChart
                                 plotlyProps={plotlyProps}
-                                theme={undefined}
+                                theme={getTheme() as any}
```

Comment:
it seems this can take in an ITheme now, so is the cast to any still needed?

Reply:
yeah, I was suprisded about needing this, it seems to be due to a version change in the FabricUI dependence, I updated the AccessibleChart library, but not the interpret dashborad to the latest version. I don't want to update our version until I have time to validate it doesn't break anything.